docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-potential-dns-tunneling-via-nslookup.asciidoc

```diff
@@ -0,0 +1,110 @@
+[[prebuilt-rule-0-16-2-potential-dns-tunneling-via-nslookup]]
+=== Potential DNS Tunneling via NsLookup
+
+This rule identifies a large number (15) of nslookup.exe executions with an explicit query type from the same host. This may indicate command and control activity utilizing the DNS protocol.
```

Comment:
```suggestion
This rule identifies a large number (15) of `nslookup.exe` executions with an explicit query type from the same host. This may indicate command and control activity utilizing the DNS protocol.
```